src/components/ImageSection.js

```diff
@@ -0,0 +1,54 @@
+import _filter from 'lodash/filter';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import { Image } from './Image';
+import { ImagesCarousel } from './ImagesCarousel';
+import { WrapperWithOrientation } from './Wrapper';
+
+export const ImageSection = ({ mediaContents }) => {
+  let images = [];
```

Comment:
i think this can be changed to `const` right, as we `push` and not reassign `images`?

Reply:
yes you are right. done in 940791c14a151ff5ce3b8ce57dbf9832b86a57ac